Makefile

```diff
@@ -87,12 +90,26 @@ tail:
 	@echo "🥫 Reading logs (Apache2, Nginx) …"
 	tail -f logs/**/*
 
-setup_incron:
-	@echo "🥫 Setting up incron jobs defined in conf/incron.conf …"
-	${DOCKER_COMPOSE} exec -T backend sh -c "\
-		echo 'root' >> /etc/incron.allow && \
-		incrontab -u root /opt/product-opener/conf/incron.conf && \
-		incrond"
+
+#----------#
+# Services #
+#----------#
+build_lang:
+	@echo "🥫 Rebuild language"
+	# Run build_lang.pl
+	${DOCKER_COMPOSE} run --rm backend perl -I/opt/product-opener/lib -I/opt/perl/local/lib/perl5 /opt/product-opener/scripts/build_lang.pl
+
+# create some bind mounted dirs, to avoid them from being owned by root
+make_base_dirs:
+	mkdir -p logs/apache2/ logs/nginx/
```

Comment:
Could you explain on why we need this, since I don't think we did before ?

Reply:
I try to explain it in the comment above. If bind mounted directories do not exist, docker-compose create them on the fly, but owned by root:root and with 0771 permissions, thus www-data won't be able to write into them (neither would be the developer, but using sudo).

Before we were using root inside container, thus this was not a problem.

Reply:
Can't we do this in the Dockerfile itself, similarly to the other dirs we create there ? And then the bind mount on `./` would mount them since they exist within the container ?